docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1223,10 +1223,12 @@ Result:
 └───────────────────────┘
 ```
 
-## FROM\_UNIXTIME
+## FROM_UNIXTIME
```

Comment:
Since `\_` seems to be used in each header to designate the `_` character, I'm not sure whether it will be better without it.

Reply:
It's legacy from the old formater. But yes, probably it's better to fix all them in another PR.